src/vs/workbench/contrib/notebook/browser/view/renderers/webviewPreloads.ts

```diff
@@ -1566,6 +1578,24 @@ async function webviewPreloads(ctx: PreloadContext) {
 				cell?.updateScroll(request);
 			}
 		}
+
+		private outputResizeHandler(cellId: string) {
+			// Debounce this callback to only happen after
+			// 250 ms. Don't need resize events that often.
+			clearTimeout(this._outputResizeTimer);
+			this._outputResizeTimer = setTimeout(() => {
+				postNotebookMessage('outputResized', {
+					cellId
+				});
+			}, 250);
+		}
+
+		private trackOutputResize(cellId: string, outputContainer: HTMLElement) {
```

Comment:
Is this a duplicate of https://github.dev/rchiodo/vscode/blob/21e64f84458090e52aede145202c9940ae21172c/src/vs/workbench/contrib/notebook/browser/view/renderers/webviewPreloads.ts#L2021-L2022 ?

Reply:
It might be repurposed to do this as well as what it was already doing though. I could try that and see if it works too

Reply:
Looks like that works too, please have another look.